fusion-cli/test/e2e/split/test.js

```diff
@@ -33,9 +45,20 @@ test('`fusion build` with dynamic imports and group chunks', async () => {
   });
   const resA = await request(`http://localhost:${port}/test-a`);
   const resB = await request(`http://localhost:${port}/test-b`);
-  const res = await request(`http://localhost:${port}/test`);
-  t.deepEqual(JSON.parse(res), [10003, 3]);
+  const resCombined = await request(`http://localhost:${port}/test-combined`);
+  const resTransitive = await request(`http://localhost:${port}/test-transitive`);
   t.deepEqual(JSON.parse(resA), [10003, 10004, 3, 4]);
   t.deepEqual(JSON.parse(resB), [10003, 10005, 3, 5]);
+  t.deepEqual(JSON.parse(resCombined), [10003, 3]);
+  t.deepEqual(JSON.parse(resTransitive), [10006, 6]);
```

Comment:
This fails on master because `"test-d"` erroneously thinks it shows up in chunks 0, 1, and 2 (10000, 10001, and 10002 in legacy build)